### DISCUSSION TITLE
Downgrade data plan grayed out

### DISCUSSION THREAD

**Discussion by sean1832** [community/community#23608] [New to GitHub]
<p>Hi, I am trying to downgrade my data plan, but am unable to do so.<br>
I have deleted all lfs storage, but the button is still greyed out.<br>
Can someone please help me here?<br>
Thanks!<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/5/c/5cb8a9796347a88292e74f5b9ee2fbc19d1020fb.png" data-download-href="/uploads/short-url/defyXhFrQK2iOr59uPc9NJ39JWP.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/84948038/181078187-10051213-126f-489a-a257-f8467a1c4138.png" alt="image" data-base62-sha1="defyXhFrQK2iOr59uPc9NJ39JWP" width="690" height="273" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/c/5cb8a9796347a88292e74f5b9ee2fbc19d1020fb_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">883×350 17.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by sean1832** [comment, marked as answer]
<p>Wops, never mind, I solved it by just changing the data pack number to 0.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/d/8da377e8565fd0d8a2076aa86bbec8612cd59f43.png" data-download-href="/uploads/short-url/kcZygJKgosNPfjH5JxPIlZ8ZcR5.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/84948038/181078190-7686e778-5292-4ff1-9b1a-df4b642716dc.png" alt="image" data-base62-sha1="kcZygJKgosNPfjH5JxPIlZ8ZcR5" width="690" height="276" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/d/8da377e8565fd0d8a2076aa86bbec8612cd59f43_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">853×342 18.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Reply by thekrocker** [reply]
lmao.. that was funny as hell, I was having anxiety about where my LFS is cached or smth! it was simple as that.. Worst UX on that..

**Comment by artyomabgaryan** [comment]
Because of lacking UX users will come to this post each time ..
BUTTON should somehow "tell" users why is it greyed out
Thanks for the question and the answer.

**Comment by kokanhan** [comment]
![2014](https://github.com/user-attachments/assets/ef9dd8a2-6d5a-449d-8e13-fc162291523b)
I have deleted my repo, but I still cannot downgrade data plan or change the data pack number to 0. Plz HLEP!

**Reply by artyomabgaryan** [reply]
You're using more than 5 gb

**Reply by Trisyd** [reply]
I'm having this same issue. I deleted the repository in question, so my total data usage is clocked at almost 4 GB, despite the component repos being considered towards my quota totaling less than 1 GB. Does this reset next month? I don't want to be charged $5 _and then_ be given the option to downgrade. Any suggestions?
![Totals](https://github.com/user-attachments/assets/6fcb5123-bd8b-43b7-9fec-892dc52710ad)
![Downgrade](https://github.com/user-attachments/assets/8e9b36ce-5915-4f7f-9b57-f44876501f4e)

**Reply by Trisyd** [reply]
I just had a similar experience to sean1832. For anyone in a similar boat, i.e. trying to downgrade data packs after just deleting a repo, wait a few minutes and then refresh your billing plans page. _Then_ go to "Remove Data Packs" and you should be given the option to select zero.
